charts/komodor-agent/templates/metrics/configmap-daemon-windows.yaml

```diff
@@ -38,6 +38,7 @@ data:
       url = {{ .Values.communications.apiServerUrl | quote }}
       url_kubelet = "https://$NODE_IP:10250"
       namespace = ""
+      poll_interval = "${INTERVAL}"
```

Comment:
where is this `INTERVAL` parameter set?

Reply:
It is set by agents-service in ResponseTelegrafIdentifyClusterConfiguration struct